full-service/src/service/account.rs

```diff
@@ -1211,7 +1191,7 @@ mod tests {
         )
         .unwrap();
 
-        assert_eq!(orphaned_txos.len(), 0);
+        assert_eq!(orphaned_txos.len(), 1);
```

Comment:
Why did this change?

Reply:
So, now that we can determine the subaddress index of a txo with only the view private and spend public keys, it allows us to assign the next subaddress index for a view only account, without having to use the offline signer. This test is finally being updated to the correct functionality